packages/statemanager/CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 1.0.1 - 2022-10-18
+
+### New EthersStateManager (experimental)
+
+There is a new dedicated state manager `EthersStateManager` added to the library. This new state manager gets its state via Ethers RPC calls and allows e.g. for a stateless execution of selected mainnet (or other Ethereum network) blocks and transactions, see PR [#2315](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2315).
+
+There are some caveats and therefore the new state manager is marked as `experimental` for now:
+
+- No `stateRoot` calculation for now, only secondary measurs like `gasUsed`
+- Performance is rather slow, particularly for more recent `mainnet` blocks
```

Comment:
```suggestion
- Performance is rather slow (when using a remote provider for state data), particularly for more recent `mainnet` blocks
```

This didn't feel true if you use a local node.  It was super speedy when I was testing against a local geth instance.